docs/data-sources/namespace.md

```diff
@@ -0,0 +1,26 @@
+# Namespace Data Source
+
+The `namespace` data source allows fetching of metadata about a given Cloudsmith namespace. The fetched data can be used to resolve permanent identifiers from a namespace's user-facing name. These identifiers can then be passed to other resources to allow more consistent identification as user-facing names can change.
+
+## Example Usage
+
+```hcl
+provider "cloudsmith" {
+    api_key = "my-api-key"
+}
+
+data "cloudsmith_namespace" "my_namespace" {
+    slug = "my-namespace"
+}
+```
+
+## Argument Reference
+
+* `slug` - (Required) The slug identifies the namespace in URIs.
```

Comment:
NAB: could be useful to bold the required arguments here and in the few places below to make quickly scanning the file easier. 

```suggestion
* `slug` - **(Required)** The slug identifies the namespace in URIs.
```